.env.test

```diff
@@ -25,5 +25,5 @@ PRIVATE_KEY_PATH=./test/setup/test-key.pem
 ATLASSIAN_URL=https://test-atlassian-instance.net
 
 # Use `trace` to get verbose logging or `info` to show less
-LOG_LEVEL=fatal
+LOG_LEVEL=info
```

Comment:
Same configuration we have in production + I need these for an integration test I added